src/main/java/frc/robot/subsystems/shooter/ShooterIOSim.java

```diff
@@ -20,6 +18,14 @@ public class ShooterIOSim implements ShooterIO {
     private final VelocityVoltage bottomControl = new VelocityVoltage(0);
     private final SimpleMotorFeedforward bottomFeedForward;
 
+    private final DutyCycleOut stop =
+            new DutyCycleOut(
+                    ShooterConstants.STOP_POWER.in(Units.RotationsPerSecond),
+                    false,
+                    true,
+                    false,
+                    false);
```

Comment:
The real Talon FX class has a convenience method called [`stopMotor()`](https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/hardware/TalonFX.html#stopMotor()) that uses the [`NeutralOut`](https://api.ctr-electronics.com/phoenix6/release/java/com/ctre/phoenix6/controls/NeutralOut.html) control request. Is it possible to use the same method here?

Reply:
I don't think so

Reply:
Okay, so just remember to use it in the real implementation.